lib/stories/animations/animations.dart

```diff
@@ -1,11 +1,21 @@
 import 'package:dashbook/dashbook.dart';
-import 'package:flame/game/game_widget.dart';
 
 import './basic.dart';
 import './aseprite.dart';
+import '../../commons/dashbook_gamewidget.dart';
 
 void addAnimationStories(Dashbook dashbook) {
   dashbook.storiesOf('Animations')
-    ..add('basic animations', (_) => GameWidget(game: BasicAnimations()))
-    ..add('aseprite', (_) => GameWidget(game: Aseprite()));
+    ..add(
+      'basic animations',
+      (_) => DashbookGameWidget(game: BasicAnimations()),
+      codeLink:
+          'https://github.com/flame-engine/flame_example/blob/master/lib/stories/animations/basic.dart',
```

Comment:
Maybe we should make the main path `https://github.com/flame-engine/flame_example/blob/master/` into a const for reuseability?